web/themes/custom/accessibility_theme/src/patterns/components/card-deck/card-deck.twig

```diff
@@ -0,0 +1,30 @@
+{{ attach_library('accessibility_theme/card-deck') }}
+
+<section class="card {{- attributes ? attributes.class -}}"
+  {{- attributes ? attributes|without(class) -}}>
+  {% if heading %}
+    {%
+      include '@accessibility_theme/heading/heading.twig' with {
+        "heading": heading
+      } only
+    %}
+  {% endif %}
+
+  <div class="card__items">
+    {% block card__items %}
+      {% for item in items %}
+        <div class="card__item">
```

Comment:
This class could be passed to the card component as part of the include below.  this way the card will print with the classes `card card__item`.